<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorChangeAction</class>
 <widget class="QWidget" name="ColorChangeAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorChangeAction</string>
  </property>
  <widget class="QPushButton" name="btnProjectConfig">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>新建 / 修改</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbWindowList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHwnd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>未绑定窗口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leProjectName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择方案</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenProject">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开方案</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>检测点数:      个</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>每</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>秒检测1次</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbFrequency">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelPointNum">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清屏</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
